src/frontend/components/LoginBanner.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import loginBanner from './loginBanner'
+import Login from './../components/Login'
+
+class LoginBanner extends Component {
+    render() {
+        return (
+        <div style={loginBanner}>
```

Comment:
Why are you using `style` prop instead of classnames of covering any related style in the scss file?

Reply:
Also, looks like you are reading css files so they shouldn't be assigned to the style attribute